src/agent/solana/exporter.rs

```diff
@@ -796,19 +803,28 @@ impl Exporter {
             network_state.blockhash,
         );
 
-        let signature = self
-            .rpc_client
-            .send_transaction_with_config(
-                &transaction,
-                RpcSendTransactionConfig {
-                    skip_preflight: true,
-                    ..RpcSendTransactionConfig::default()
-                },
-            )
-            .await?;
-        debug!(self.logger, "sent upd_price transaction"; "signature" => signature.to_string(), "instructions" => instructions.len(), "price_accounts" => format!("{:?}", price_accounts));
-
-        self.inflight_transactions_tx.send(signature).await?;
+        let tx = self.inflight_transactions_tx.clone();
+        let logger = self.logger.clone();
+        let rpc_client =
+            RpcClient::new_with_timeout(self.rpc_url.clone().to_string(), self.rpc_timeout);
```

Comment:
I think it is slightly unfortunate that you are storing the url / timeout instead of the RpcClient above so that you can use `move` in the spawn call below. There is almost certainly a better way to do this -- you may be able to use`Arc`? David will know for sure.

Reply:
Right. I thought the rpc request would be mutable for some reason but it is immutable and Arc works fine. Thanks for pointing it out.